extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -259,6 +259,14 @@ export function findQueryLogFile(resultPath: string): string {
   return path.join(resultPath, 'query.log');
 }
 
-export function findQueryStructLogFile(resultPath: string): string {
+export function findQueryEvalLogFile(resultPath: string): string {
   return path.join(resultPath, 'evaluator-log.jsonl');
 }
+
+export function findQueryEvalLogSummaryFile(resultPath: string): string {
+  return path.join(resultPath, 'evaluator-log.jsonl.summary');
+}
+
+export function findQueryEvalLogEndSummaryFile(resultPath: string): string {
+  return path.join(resultPath, 'eoq-evaluator-log.jsonl.summary');
```

Comment:
Also here:

```suggestion
  return path.join(resultPath, 'eoq-evaluator-log.summary.jsonl');
```

Reply:
Same as above, it's not a JSONL file but rather a condensed version of the summary text file. I didn't know how to name it so appended `eoq` to the beginning of the file rather than at the end, but

`evaluator-log.jsonl.summary.eoq` might also work?

Reply:
Maybe `evaluator-log-end.summary`? As I understand it, this is only used internally so I don't suppose it matters too much.